### PR TITLE
Move documentation into its own folder

### PR DESCRIPTION
**Pull request by TheLimeGlass**
### Description
Move documentation into it's own folder read more at issue

---
**Related Issues:** https://github.com/SkriptLang/skript-docs/issues/34

### CODE REVIEW

File: src/main/java/ch/njol/skript/doc/Documentation.java [hunk 1]
```diff
@@ -60,8 +60,8 @@ public class Documentation {
 	private static final Pattern CP_EMPTY_PARSE_MARKS_PATTERN = Pattern.compile("\\(\\)");
 	private static final Pattern CP_PARSE_TAGS_PATTERN = Pattern.compile("(?<=[(|\\[ ])[-a-zA-Z0-9!$#%^&*_+~=\"'<>?,.]*?:");
 	private static final Pattern CP_EXTRA_OPTIONAL_PATTERN = Pattern.compile("\\[\\(((\\w+? ?)+)\\)]");
-	private static final File DOCS_TEMPLATE_DIRECTORY = new File(Skript.getInstance().getDataFolder(), "doc-templates");
-	private static final File DOCS_OUTPUT_DIRECTORY = new File(Skript.getInstance().getDataFolder(), "docs");
+	private static final File DOCS_TEMPLATE_DIRECTORY = new File(Skript.getInstance().getDataFolder(), "documentation/doc-templates");
+	private static final File DOCS_OUTPUT_DIRECTORY = new File(Skript.getInstance().getDataFolder(), "documentation/docs");
```
Comment (AyhamAl-Ali) [inline src/main/java/ch/njol/skript/doc/Documentation.java:64]:
```suggestion
	private static final File DOCS_TEMPLATE_DIRECTORY = new File(Skript.getInstance().getDataFolder(), "documentation/templates");
	private static final File DOCS_OUTPUT_DIRECTORY = new File(Skript.getInstance().getDataFolder(), "documentation/output");
```

**Review by AyhamAl-Ali — CHANGES_REQUESTED** [review]
As you suggested on skript-docs

**Review by AyhamAl-Ali — APPROVED** [review]

**Review by Moderocky — APPROVED** [review]

**Comment by TheLimeGlass** [comment]
@Pikachu920 or @AyhamAl-Ali are to merge, as the docs bot on the documentation repository needs to be updated.

**Review by APickledWalrus — APPROVED** [review]

File: src/main/java/ch/njol/skript/doc/Documentation.java [hunk 2]
```diff
@@ -60,8 +60,8 @@ public class Documentation {
 	private static final Pattern CP_EMPTY_PARSE_MARKS_PATTERN = Pattern.compile("\\(\\)");
 	private static final Pattern CP_PARSE_TAGS_PATTERN = Pattern.compile("(?<=[(|\\[ ])[-a-zA-Z0-9!$#%^&*_+~=\"'<>?,.]*?:");
 	private static final Pattern CP_EXTRA_OPTIONAL_PATTERN = Pattern.compile("\\[\\(((\\w+? ?)+)\\)]");
-	private static final File DOCS_TEMPLATE_DIRECTORY = new File(Skript.getInstance().getDataFolder(), "doc-templates");
-	private static final File DOCS_OUTPUT_DIRECTORY = new File(Skript.getInstance().getDataFolder(), "docs");
+	private static final File DOCS_TEMPLATE_DIRECTORY = new File(Skript.getInstance().getDataFolder(), "documentation/templates");
+	private static final File DOCS_OUTPUT_DIRECTORY = new File(Skript.getInstance().getDataFolder(), "documentation/output");
```
Comment (AyhamAl-Ali) [inline src/main/java/ch/njol/skript/doc/Documentation.java:64]:
```suggestion
	private static final File DOCS_TEMPLATE_DIRECTORY = new File(Skript.getInstance().getDataFolder(), "docs/templates");
	private static final File DOCS_OUTPUT_DIRECTORY = new File(Skript.getInstance().getDataFolder(), "docs");
```

**Review by AyhamAl-Ali — CHANGES_REQUESTED** [review]

**Review by AyhamAl-Ali — APPROVED** [review]

**Pull request merged**